pkg/factory/risingwave_object_factory.go

```diff
@@ -1541,6 +1542,14 @@ func (f *RisingWaveObjectFactory) argsForStandalone() []string {
 	}
 }
 
+func (f *RisingWaveObjectFactory) argsForStandaloneV2() []string {
+	return []string{
+		"--config-path=$(RW_CONFIG_PATH)",
```

Comment:
I believe we also need to configure meta's dashboard port.

Reply:
Can it be done with env var?

Reply:
Sure if there's such an env var.

Reply:
Could you add the env to the container? So that the dashboard port will match the default value.